tests/src/JIT/Directed/tailcall/more_tailcalls.il

```diff
@@ -0,0 +1,3483 @@
+
+//  Microsoft (R) .NET Framework IL Disassembler.  Version 4.5.30319.0
+
+
```

Comment:
It is useful to keep the C# file that I assume you have used to generate this for reference. It is hard to understand and review what the pile of IL does otherwise.

Reply:
Should I just add it as a comment at the top? It is also relying on InlineIL.Fody, but I guess that is ok?

Reply:
Comment at the top or separate file. Either way is fine.